packages/babel-types/src/ast-types/generated/index.ts

```diff
@@ -1542,6 +1544,7 @@ export interface ClassPrivateProperty extends BaseNode {
   value: Expression | null;
   decorators: Array<Decorator> | null;
   static: any;
+  typeAnnotation: TypeAnnotation | TSTypeAnnotation | Noop | null;
```

Comment:
`typeAnnotation` is an optional property: Babel does not assign it to AST unless `flow` or `typescript` plugin are enabled.

Reply:
added the same definition as in `ClassProperty`